website/docs/components/form/super-select/partials/specifications/states.md

```diff
@@ -0,0 +1,23 @@
+## States
+
+!!! Info
+
+**A note on disabled states**
+
+Because disabled states completely remove the interactive function of an element, it can be challenging for a user to understand why it has been disabled and/or why they cannot interact with that element. In an effort to avoid this confusion, we opt for using methods like displaying or hiding the element and, thus, are not offering disabled states for the Dropdown. Read more about [when to enable vs hide](https://hashicorp.atlassian.net/wiki/spaces/DES/pages/2678685874/Hiding+Disabling).
```

Comment:
If I'm not mistaken, we are talking about the list for the SuperSelect. When the SuperSelect is disabled, we do not allow a disabled list to be shown (because you cannot open it in that state thus it's not available). So we should probably reword this to say "SuperSelect list" vs "dropdown."


```suggestion
Because disabled states completely remove the interactive function of an element, it can be challenging for a user to understand why it has been disabled and/or why they cannot interact with that element. In an effort to avoid this confusion, we opt for using methods like displaying or hiding the element and, thus, are not offering disabled states for the SuperSelect list. Read more about [when to enable vs hide](https://hashicorp.atlassian.net/wiki/spaces/DES/pages/2678685874/Hiding+Disabling).
```

Reply:
@majedelass I think the wording is still a bit confusing.

The SuperSelect as a whole can be disabled which makes the "trigger" non-interactive so when you click on it nothing happens (ie the dropdown with the list of options will not display.)

Individual option items can also be disabled. This means that when you click on them within the dropdown list nothing happens. It is still possible to set the disabled options to be pre-selected however. (Alex just updated the Showcase to include examples of these and I fixed some related styles.)

Reply:
Maybe we should reword this to just say: 

```suggestion
Because disabled states completely remove the interactive function of an element, it can be challenging for a user to understand why it has been disabled and/or why they cannot interact with that element. In an effort to avoid this confusion, we opt for using methods like displaying or hiding the element. Read more about [when to enable vs hide](https://hashicorp.atlassian.net/wiki/spaces/DES/pages/2678685874/Hiding+Disabling).
```

Reply:
Thanks @majedelass this looks better. Applying...